docs/page.schema.json

```diff
@@ -77,6 +77,18 @@
                                                         "s": {
                                                             "type": "integer",
                                                             "description": "A style ID as defined in issue.schema.json/s"
+                                                        },
+                                                        "gn": {
+                                                            "type": "boolean",
+                                                            "description": "Indicates whether the token is/should be followed by a whitespace. `gn==true` means no white space is required (`gn` stands for 'glue next'). When not specified it is assumed to be `false`."
+                                                        },
+                                                        "hy": {
+                                                            "type": "boolean",
+                                                            "description": "Indicates whether the token constitutes the first part of a hyphenated word. When not specified it is assumed to be `false`."
```

Comment:
Do we record the hyphen itself as well or is it implicit similiar to the whitespace? We have some words with multiple hyphenation due to limited space in a table cell. How do i deal with this? Should I record all? Accordingly, I suggest to write "the former part before the hyphen (_incl. / excl._ hyphen)" instead of "first".

Reply:
that's an interesting case that we haven't had so far. I would mark the first part with `hy=True`, and all the remaining with `nf=...`. 

What do you think @simon-clematide and @e-maud ?

Anyway, we will have to test carefully the behavior of the rebuilt script in such cases.